### ISSUE TITLE
Please add a filter arguments in the alm_query_posts()

### ISSUE THREAD

**Issue by mihdan** [dcooney/wordpress-ajax-load-more#86]
```php
// Set current page number for determining item number		
   		if($page == 0){
            $alm_page_count = 1;
   		}else{   		
   		   $alm_page_count = $page + 1;
   		}   		
// Here   
$args = apply_filter( 'alm_query_posts_args', $args );

   		// WP_Query()
   		$alm_query = new WP_Query( $args );
```

**Comment by mihdan** [comment]
?

**Comment by dcooney** [comment]
:+1: soon

**Comment by dcooney** [comment]
Better late than never?
Included with 2.8.6.
https://connekthq.com/plugins/ajax-load-more/docs/filter-hooks/

**Comment by mihdan** [comment]
Thx
